Makefile

```diff
@@ -33,9 +24,9 @@ dapp:
 
 kevm:
 	git submodule update --init --recursive -- deps/evm-semantics
-	cd deps/evm-semantics/                                                \
-	    && make deps RELEASE=true SKIP_HASKELL=true SKIP_LLVM=true        \
-	    && make build-java RELEASE=true -j4 JAVA_KOMPILE_OPTS=--emit-json
+	cd deps/evm-semantics/                               \
+	    && make deps RELEASE=true SKIP_HASKELL=true      \
```

Comment:
I `LLVM` no longer skippable? Or do we need it now?

Reply:
It's no longer skippable. There are at least 2 reasons, and I'm not sure if they interact. Specifically:

-   The new `krun` pipeline uses the LLVM backend all the time for macro expansion, _even_ if you're then going to use the Haskell backend for symbolic execution.
-   The new KEVM package cannot be built without the LLVM backend, because of how the installer works.